neptune/query/query.go

```diff
@@ -28,6 +28,9 @@ const (
 	CodeExists = `g.V().hasLabel('_code_list')` +
 		`.has('listID', '%s').has('edition', '%s')` +
 		`.in('usedBy').has('value', "%s").count()`
+	GetUsedByEdge = `g.V().hasId('_code_%s_%s')` +
+		`.outE('usedBy')` +
+		`.where(otherV().hasLabel('_code_list').has('_code_list', 'listID', '%s'))`
```

Comment:
I think the ID of the code list node can be used for lookup, instead of the label and property. 

I realise in the ID there is also the edition. I think we may need to know the edition of the code list when looking it up, else we may get multiple results for a code list if there are multiple editions.

See an example ID for CPIH aggregate code list here: https://github.com/ONSdigital/dp-code-list-scripts/blob/master/code-list-scripts/gremlin/cpih1dim1aggid.grm#L1

Reply:
thanks! I will add the edition, and will use node Id instead of `hasLabel` for codeList node

Reply:
I tried the following query:
```
g.V().hasId('_code_{codeListID}_{code}').outE('usedBy').where(otherV().hasId('_code_list_{codeListID}_{edition}'))
```
And it works, but we need to pass the edition to the dimension importer (it currently doesn't have this information)

Reply:
Reponse from @eldeal regarding this:
```
There is a whole chunk of work we need to do around getting the import process to understand editions in code lists. I think its out of scope for the currect epic, so I would not worry about it for now
```
I think we should continue using the label, but I will leave a comment so that when editions is properly implemented in code lists we change this query.